src/System.ComponentModel.Composition/src/System.ComponentModel.Composition.csproj

```diff
@@ -5,6 +5,7 @@
     <ProjectGuid>{2D694AC8-A12F-4622-9405-74E20EC40C3A}</ProjectGuid>
     <RootNamespace>System.ComponentModel.Composition</RootNamespace>
     <AssemblyName>System.ComponentModel.Composition</AssemblyName>
+    <DefineConstants>$(DefineConstants);FEATURE_TRACING</DefineConstants>
```

Comment:
I don't think we need to define this we should just include only the branches of code where this was defined.

Reply:
I figured it might be handy while we get the tests passing (undefine it and you get debug output)